bin/component-playground.js

```diff
@@ -15,4 +16,4 @@ var server = new WebpackDevServer(compiler, {
   hot: true
 });
 
-server.listen(8989, 'localhost', function() {});
+server.listen(userConfig.server.port, userConfig.server.hostname, function() {});
```

Comment:
Oops, sorry for that. I've configured my atom with jscs linter plugin to avoid this in the future.